samples/word/03-content-controls/insert-and-change-content-controls.yaml

```diff
@@ -7,82 +7,92 @@ api_set:
     WordApi: 1.1
 script:
     content: |-
-        $("#insert-controls").click(insertContentControls);
-        $("#change-controls").click(modifyContentControls);
-        $("#setup").click(setup);
+        $("#insert-controls").click(() => tryCatch(insertContentControls));
+        $("#change-controls").click(() => tryCatch(modifyContentControls));
+        $("#setup").click(() => tryCatch(setup));
 
 
-        function insertContentControls() {
+        async function insertContentControls() {
             //traverses each paragraph of the document and wraps a content control on each with either a even or odd tags
-            Word.run(function (context) {
-                var paragraphs = context.document.body.paragraphs;
-                context.load(paragraphs);
-                return context.sync()
-                    .then(function () {
-                        for (var i = 0; i < paragraphs.items.length; i++) {
-                            console.log(paragraphs.items.length);
-                            var contentControl = paragraphs.items[i].insertContentControl();
-                            //for even we tag "even"
-                            if (i % 2 == 0)
-                                contentControl.tag = "even";
-
-                            else
-                                contentControl.tag = "odd";
-                        }
-                        return context.sync();
-                    })
-            })
-                .catch(OfficeHelpers.Utilities.log);
+            await Word.run(async (context) => {
+                let paragraphs = context.document.body.paragraphs;
+                paragraphs.load();
```

Comment:
paragraphs.load();  is another empty load.

Reply:
local fixed.